modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated-comparison/tests

```diff
@@ -4,6 +4,7 @@
   [nonlinear]
     type = 'Exodiff'
     input = segregated-nonlinear.i
+    should_crash = true
```

Comment:
```suggestion
    # only testing two iterations
    should_crash = true
```
same for the others

Reply:
Added as suggested.

Reply:
these tests should be doing this instead
```suggestion
    cli_args = 'Executioner/continue_on_max_its=true'
```